### ISSUE TITLE
[FEATURE] Add accessibility tests with Playwright

### ISSUE THREAD

**Issue by YuriDevAT** [YurisCodingClub/accessibility-mentor#15]
As a developer I want to get notified if my contribution fails accessibility with automated testing via Playwright.

**Acceptance Criteria**
- [ ] Install the latest version of Playwright https://playwright.dev/
- [ ] Add the official a11y basic test by deque https://playwright.dev/docs/accessibility-testing

**Comment by Pushkarm029** [comment]
hello @YuriDevAT can you assign this issue to me?

**Comment by Pushkarm029** [comment]
@YuriDevAT please review

**Comment by Chizycodes** [comment]
Can you assign this to me? @YuriDevAT

**Comment by YuriDevAT** [comment]
Sure @Chizycodes Glad to have you contributing to the project 😄
